pkg/controller/syncsetinstance/syncsetinstance_controller_test.go

```diff
@@ -1539,3 +1539,16 @@ func TestSSIReApplyDuration(t *testing.T) {
 		}
 	}
 }
+
+func TestFilterApplyError(t *testing.T) {
+	err1 := "error when creating \"/tmp/apply-475927931\": namespaces \"openshift-am-config\" not found"
+	expectedFilteredErr1 := "namespaces \"openshift-am-config\" not found"
+	if filterApplyError(err1) != expectedFilteredErr1 {
+		t.Fatalf("expected temporary file to be trimmed from error message")
```

Comment:
This should use `t.Errorf` instead. You want the second part of the test to run even when the first part fails.

Reply:
Changed to `t.Errorf`.